exp/anbox-cloud.md

```diff
@@ -31,62 +31,82 @@ See the following table for a comparison of features for the different variants:
+* **AMS node controller** – The AMS node controller puts the appropriate firewall rules in place when a container is started or stopped to control ingress and egress traffic.
+
+Outside the Anbox subcluster, you have the following machines:
+
+  * **Juju controller**, which controls how Anbox Cloud is deployed and managed.
+  * **Anbox Application Registry (AAR)**, which provides a central repository for applications created on Anbox Cloud. Using an AAR is very useful for larger deployments involving multiple subclusters, in order to keep applications in sync. This component is not mandatory, but strongly recommended in a production deployment, to keep applications in sync across the different subclusters.
+
+### Streaming stack
+
+The diagram below depicts the streaming stack along with the core stack and user specific components:
 
 ![Anbox Cloud streaming stack|690x440](https://assets.ubuntu.com/v1/bcf90bb6-anbox_cloud_streaming-stack.png)
 
-The main components powering the streaming stack in Anbox Cloud are:
+When the streaming stack is in use, each Anbox subcluster has the following additional components:
 
-**Agent**: Software running on a server equipped with a GPU connected to Anbox Cloud. It serves as an entry point that the gateway can connect to.
+* **TURN/STUN servers** - Servers that find the most optimal network path between a client and the Anbox container running its application. The streaming stack provides secure STUN and TURN servers, but you can use public ones as well.
+  
+* **Stream agent** - Serves as an entry point that the gateway can connect to to talk to the AMS of the Anbox subcluster.
 
-**Anbox Stream Gateway**: The central component that connects clients with agents. Its role is to choose the best possible region depending on the user location and server capacities.
+You also need an additional machine to host the streaming stack control plane with the following components:
 
-**Client**: The end user application that will display the stream. It can be a desktop application, a website, a mobile application, a TV, a car system or anything capable of handling a WebRTC stream. Anbox Cloud provides an SDK along with the streaming stack to simplify integration with web-based applications.
+  * **Stream gateway** - The central component that connects clients with agents. Its role is to choose the best possible subcluster depending on the user location and server capacities.
+  * **NATS** - A messaging system that the different components use to communicate. NATS is  responsible for the communication between the stream gateway and the stream agent. For more information, see [NATS protocol](https://docs.nats.io/reference/reference-protocols/nats-protocol).
+	
+	[note type="information" status="Note"] Currently, these two components must actually be hosted on different machines because they have different operating system requirements.[/note]
```

Comment:
Should this note be indented? I think I'd put it on the main level, not as part of the list.

Reply:
Thanks, I have a bit of trouble figuring out which lines are indented in my editor preview.